### ISSUE TITLE
TShock Issue

### ISSUE THREAD

**Issue by devstark03** [Pryaxis/TShock#2921]
* TShock version: 4.5.4
* TShock build number (if known):

#### Reproduction steps (if applicable)?

1. Start server
2. Wait five minutes
3. Warning message appears and continues to appear every five minutes, thus flooding my logs and console over time.

#### Any stack traces or error messages (if known)?

```
Terraria Server v1.4.2.3

Listening on port 3141
Type 'help' for a list of commands.

Login before join enabled. Users may be prompted for an account specific password instead of a server password on connect.
Login using UUID enabled. Users automatically login via UUID.
A malicious server can easily steal a user's UUID. You may consider turning this option off if you run a public server.
: Server started

: UpdateManager warning: Error: NameResolutionFailure
Retrying in 5 minutes.

: UpdateManager warning: Error: NameResolutionFailure
Retrying in 5 minutes.

: UpdateManager warning: Error: NameResolutionFailure
Retrying in 5 minutes.
```

It's not a critical issue, but it is very annoying and I can't find an answer anywhere on the internet. Searched through configs for anything relating to updates but couldn't find anything relevant.

**Comment by pcider** [comment]
Your TShock version is (probably) no longer supported. The latest version is v5.1.3.

**Comment by devstark03** [comment]
My eyes widened as I wondered how on earth I got so far behind on updates. Tried upgrading and remembered why. The way the server files are set up between the two versions is different to the point that my custom scripts stopped working on newer TShock versions and so I put off the upgrade until I eventually forgot. Did a complete overhaul on my system just now, and the latest update (5.1.3) has solved the original issue. Thank you.

Edit: updating can provide a temporary solution, but the real underlying issue in my circumstance was not TShock itself, but my DNS failing to resolve hostnames from URLs, which is an entirely separate issue that I fixed.
